sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1225,6 +1225,12 @@ object QueryExecutionErrors {
     new RuntimeException("Remote operations not supported")
   }
 
+  def missingMethodForTimestampWithoutTZ(method: String): Throwable = {
+    new IllegalStateException(
```

Comment:
This is not a user-facing error and we should just inline it, similar to `UnresolvedAttribute.exprId`